app/models/user.rb

```diff
@@ -1,4 +1,25 @@
 class User < ActiveRecord::Base
   has_many :bids
-  validates :email, email: true, allow_blank: true
+
+  validates :email, allow_blank: true, email: true
+  validate :validate_credit_card_form
+
+  private
+
+  def validate_credit_card_form
+    return if credit_card_form_url.blank?
+
+    uri = URI.parse(credit_card_form_url)
+
+    case uri
+    when URI::HTTPS
+    # nothing to do here
+    when URI::HTTP
+      errors.add(:credit_card_form_url, 'must use HTTPS')
```

Comment:
I think it's worth doing. But it also means we should let vendors know way in advance. We could also relax the rule for now, let the vendors know, and in a month or so, put the rule in place.

We might also want to consider not preventing someone from bidding if they don't have that URL. We just make it clear we won't be able to pay them until a proper URL is submitted.